docs/aws/README.md

```diff
@@ -103,17 +100,18 @@ kind: Provisioner
 metadata:
   name: default
 spec:
+  ttlSeconds: 30
```

Comment:
I added this just to show how people can configure it and for the cleanup section later to help people avoid costs.